frontend/src/components/Profile/ProfileAbout.js

```diff
@@ -0,0 +1,35 @@
+import {Button, Empty} from 'antd';
+import { Card } from 'antd';
+import {Context} from "../../store";
+import {useContext} from "react";
+
+
+const ProfileAbout = ({about, username}) => {
+
+    const [state, dispatch] = useContext(Context);
+
+    console.log(state.auth.user.username)
```

Comment:
We should probably remove this at some point :)

Reply:
Removed in later PR-s :)